app/services/locale_ninja/github_api_service.rb

```diff
@@ -3,27 +3,44 @@
 module LocaleNinja
   class GithubApiService
     REPOSITORY_FULLNAME = Rails.application.credentials.github.repository_name
-    private_constant :REPOSITORY_FULLNAME
+    TRANSLATIONS_SUFFIX = '__translations'
+
+    public_constant :REPOSITORY_FULLNAME
+    public_constant :TRANSLATIONS_SUFFIX
 
     def initialize(access_token:)
       @client = ::Octokit::Client.new(access_token:)
     end
 
     attr_reader :client
 
-    def user
-      @client.user
+    delegate :user, :repository, :update_contents, :create_contents, :ref, :content, :blob, :tree, to: :@client
```

Comment:
I would suggest 
```Ruby
delegate :user, to: :@client
delegate :update_contents, :create_contents, :ref, :content, :blob, :tree, to: :@client, private: true
```
as we don't want to interact with github another way than in this file